### PR TITLE
Updates for awswrangler 2.12.0

### PR DESCRIPTION
**Pull request by mratford**
Needs more test coverage.

### CODE REVIEW

File: pydbtools/utils.py [hunk 1]
```diff
@@ -122,8 +123,34 @@ def get_database_name_from_userid(user_id: str) -> str:
...
+        # Return the first database seen in the
+        # form "database.table"
+        xs = table.split(".")
+        if len(xs) > 1:
+            return xs[0]
+
+    # Return default in case of failure to parse
+    return None
+    # Fails tests but might give fewer errors...
+    # return "unknown_database"
```
Comment (isichei) [inline pydbtools/utils.py:148]: remove this comment

File: pydbtools/_wrangler.py [hunk 2]
```diff
@@ -98,27 +98,43 @@ def wrapper(*args, **kwargs):
             # Set s3 to default s3 path
             argmap["s3_output"] = s3_output
 
-        # Set ctas_approach to False if not set
+        # Set ctas_approach to True if not set
```
Comment (isichei) [inline pydbtools/_wrangler.py:101]:
```suggestion
        # Set ctas_approach to True if not set.
        # Although awswrangler does this by default, we want to ensure
        # that timestamps are read in correctly to pandas using pyarrow.
        # Therefore forcing the default option to be True incase future versions
        # of wrangler change their default behaviour. It also clarifies our intent.
```

File: pydbtools/utils.py (hunk 1 above)
Comment (mratford) [reply pydbtools/utils.py]: https://github.com/moj-analytical-services/pydbtools/pull/57/commits/2cc5bb444212ad8f42bb68fddf8ef023ceb4d1b8

**Review by isichei — APPROVED** [review]
LGTM - minor comment and suggestion. But happy to merge to develop.

**Pull request merged**